evalml/data_checks/invalid_targets_data_check.py

```diff
@@ -89,6 +89,12 @@ def validate(self, X, y):
                                                      data_check_name=self.name,
                                                      message_code=DataCheckMessageCode.TARGET_BINARY_NOT_TWO_UNIQUE_VALUES,
                                                      details=details).to_dict())
+        if self.problem_type == ProblemTypes.MULTICLASS and value_counts.min() <= 1:
```

Comment:
If I'm understanding this correctly, this should only trigger for values that just have one sample right? (0 samples wouldn't be a class); in that case, maybe it's not really useful to return `least_populated_class_count` since that's always 1?

Reply:
Yep, you're right. I was over complicating it. I removed the `least_populated_class_count` from details.